docs/config/cluster.rst

```diff
@@ -356,6 +356,27 @@ new master node is elected.
    explicitly list the names or IP addresses of the master-eligible nodes whose
    votes should be counted in the very first election.
 
+.. _conf_discovery_type:
+
+**discovery.type**
+  | *Default:* ``zen``
+  | *Runtime:* ``no``
+  | *Allowed Values:*   ``zen | single-node``
+
+  Specifies whether CrateDB should form a multiple-node cluster. By default,
+  CrateDB discovers other nodes when forming a cluster and allows other nodes to
+  join the cluster later. If ``discovery.type`` is set to ``single-node``,
+  CrateDB forms a single-node cluster and the node won't join any other
+  cluster. This may be usefull mainly for testing purposes which requires the
```

Comment:
```suggestion
  cluster. This can be useful for testing. It is not recommend to use this for production setups. The ``single-node`` mode also skips bootstrap checks.
```

Maybe with cross-reference to the bootstrap checks docs